frontend/test/e2e/query_builder/query_builder.spec.js

```diff
@@ -63,7 +63,7 @@ describeE2E("query_builder", () => {
             await d.select("#CreateDashboardModal .Button.Button--primary").wait().click().waitRemoved(); // wait for the modal to be removed
             incrementDashboardCount();
 
-            await d.waitUrl(getLatestDashboardUrl() + "?add=" + cardId);
+            await d.waitUrl(getLatestDashboardUrl() + "#add=" + cardId);
```

Comment:
Since both this and the AddToDashSelectModal need this `#add` format with an ID would it be worth extracting that somehow so in the future if we need to change it we don't need to update it two places?